release/xgboost_tests/app_config_gpu.yaml

```diff
@@ -1,10 +1,4 @@
 base_image: {{ env["RAY_IMAGE_ML_NIGHTLY_GPU"] }}
-env_vars:
-  # Manually set NCCL_SOCKET_IFNAME to "ens" so NCCL training works on
-  # anyscale_default_cloud.
-  # See https://github.com/pytorch/pytorch/issues/68893 for more details.
-  NCCL_SOCKET_IFNAME: ens
```

Comment:
What's the default value if we don't explicitly set `NCCL_SOCKET_IFNAME`?

Reply:
The default behavior is described here: https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/env.html#nccl-socket-ifname